client.bal

```diff
@@ -383,15 +383,15 @@ public client class Client {
 
 # Configuration record for NetSuite
 #
-# + accountId - NetSuite account Id  
-# + consumerSecret - Netsuite Integration App consumer secret
-# + baseURL - Netsuite baseURL for web services and this "/services/NetSuitePort_2020_2" should be added to the link.   
-# + consumerId - Netsuite Integration App consumer Id   
+# + accountId - NetSuite Account ID  
+# + consumerSecret - Netsuite Integration application consumer secret
+# + baseURL - Netsuite SuiteTalk baseURL for SOAP web services  
```

Comment:
We are only asking for base URL. I think it will be useful for the user if we give him an idea what the base url is taking an example url. We can add that information here itself and it will appear as a tooltip.

Reply:
Updated with an example.